src/devrules/core/git_service.py

```
@@ -448,3 +417,100 @@ def get_current_repo_name() -> str:
+
+
+def get_default_branch() -> str:
+    """Get the default branch (main or master).
+
+    Returns:
+        Default branch name
+    """
+    try:
+        # Try to get from remote
+        result = subprocess.run(
+            ["git", "symbolic-ref", "refs/remotes/origin/HEAD"],
```

Ruff (0.14.13)

Starting a process with a partial executable path

(S607)